tina/collectionsSchema/navigationBar.tsx

```diff
@@ -0,0 +1,134 @@
+const modals = ['BookDemo', 'EmailForm'];
+const iconOptions = ['FaCalendarDay', 'MdEmail'];
+
+export const navigationBarCollection = {
+  label: 'Navigation Bar',
+  name: 'navigationBar',
+  path: 'content/navigationBar',
+  format: 'json',
+  fields: [
```

Comment:
Since there is only ever going to be 1 nav menu, this can be a Single Document Collection
https://tina.io/docs/editing/single-document-collections/

Reply:
✅ and updated docs with example